src/ynEIGEN/ynEigen.sol

```diff
@@ -13,6 +13,7 @@ import {ynBase} from "src/ynBase.sol";
 
 interface IynEigenEvents {
     event Deposit(address indexed sender, address indexed receiver, uint256 amount, uint256 shares);
+    event DepositAsset(address indexed sender, address indexed receiver, address indexed asset, uint256 amount, uint256 shares);
```

Comment:
Can this Deposit event be removed?

Reply:
yep remove it

Reply:
yes it can no longer used in the solidity code

it does still exist on-chain already though. but we can remove it.